astropy/wcs/wcsapi/low_level_api.py

```diff
@@ -257,16 +258,38 @@ def pixel_bounds(self):
         """
         return None
 
+    @property
+    def pixel_axis_names(self):
+        """
+        An iterable of strings describing the name for each pixel axis.
+
+        If an axis does not have a name, an empty string should be returned.
```

Comment:
Maybe this should be reworded to:
```suggestion
        If an axis does not have a name (the default if a subclass does not
        override), an empty string should be returned.
```
Or similar to get at the point that this is *optional*, and if you do nothing everything still works?